gh-pages/_docs/06-05-suspicious-metrics.md

```diff
@@ -0,0 +1,14 @@
+---
+
+permalink: /docs/suspicious-metrics/
+title: "Suspicious metrics"
+
+This feature compares the values of different metrics with metric values of 241 Open Source Java projects.
+Based on this data suspicious and inconspicuous metrics are identified and corresponding suggestions can be clicked to view them.
+
+Be aware that metrics for other programming languages might not be comparable to Java Reference metric values.
+
+To activate this function it is necessary to open the global configuration menu and activate the experimental features,
+after that the Suspicious button will appear in the main menu. Pressing this button will open the suspicious metrics panel.
+Here are listed the suspicious metrics, you can change the map view with the suspicious metrics buttons, you can also see
+highlighted on the map the files with a high risk factor (if any).
```

Comment:
Maybe try rewording the last sentence. Something like "Listed you will find the suspicious metrics, clicking one of them highlights high-risk files directly on the map."